components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/utils/CmdUtils.java

```diff
@@ -228,6 +228,32 @@ public static String getCLIHome() {
         return System.getProperty(CliConstants.CLI_HOME);
     }
 
+    /**
+     * Get cacerts location
```

Comment:
When adding java docs, make sure to add a full stop at the end of first sentence. This is validated by checkstyle.
```suggestion
     * Get cacerts location.
```